dts: edtlib: Error running devicetree tests

### Describe the bug

Commit f55358bcbfcfc4e7bd7d5c00bf6a16a13f01e8f6  introduced an issue for EDTLIB, bisecting

```
f55358bcbfcfc4e7bd7d5c00bf6a16a13f01e8f6 is the first bad commit
commit f55358bcbfcfc4e7bd7d5c00bf6a16a13f01e8f6 (HEAD)
Author: Kyle Bonnici <kylebonnici@hotmail.com>
Date:   Tue Dec 16 13:36:04 2025 +0100

    dts: use lowercase hex values in DTS files.

    Apply chnages from dts-linter 0.3.9.

    Improve compliance with DTS Coding Style which says that:

    4) Hex values in properties, e.g. “reg”, shall use lowercase hex.
       The address part can be padded with leading zeros.

    Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>
```

### Regression

- [x] This is a regression.

### Steps to reproduce

```shell
$ pip install tox
$ cd scripts/dts/python-devicetree
$ tox
```

### Relevant log output

```shell
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- Captured log call -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
WARNING  devicetree.edtlib:edtlib.py:1778 'oldprop' is marked as deprecated in 'properties:' in test-bindings/deprecated.yaml for node /test-deprecated.
WARNING  devicetree.edtlib:edtlib.py:2577 unit address and first address in 'reg' (0x1) don't match for /reg-zero-size-cells/node
WARNING  devicetree.edtlib:edtlib.py:2577 unit address and first address in 'reg' (0x5) don't match for /reg-ranges/parent/node
WARNING  devicetree.edtlib:edtlib.py:2577 unit address and first address in 'reg' (0x30000000200000001) don't match for /reg-nested-ranges/grandparent/parent/node
WARNING  devicetree.edtlib:edtlib.py:2640 compatible 'enums' in binding 'test-bindings/enums.yaml' has non-tokenizable enum for property 'string-enum': 'foo bar', 'foo_bar'
WARNING  devicetree.edtlib:edtlib.py:2646 compatible 'enums' in binding 'test-bindings/enums.yaml' has enum for property 'tokenizable-lower-enum' that is only tokenizable in lowercase: 'bar', 'BAR'
================================================================================================================================================================================== short test summary info ==================================================================================================================================================================================
FAILED tests/test_edtlib.py::test_interrupts - devicetree.edtlib.EDTError: component 'node@70000000E' in path '/interrupt-map-bitops-test/node@70000000E' does not exist
=============================================================================================================================================================================== 1 failed, 87 passed in 0.41s ================================================================================================================================================================================
py3: exit 1 (0.56 seconds) /home/pdgendt/zephyrproject/zephyr/scripts/dts/python-devicetree> python -m pytest tests pid=38636
  py3: FAIL code 1 (3.04=setup[2.48]+cmd[0.56] seconds)
  evaluation failed :( (3.06 seconds)
```

### Impact

Functional Limitation – Some features not working as expected, but system usable.

### Environment

- OS: Ubuntu 24.04
- Toolchain: Zephyr SDK 0.17.4
- Commit: a8f8f7da3357fa86fd87ad7d6f84a121404aff0b
- Python: 3.12.12

### Additional Context

_No response_

--- a/scripts/dts/python-devicetree/tests/test_edtlib.py
+++ b/scripts/dts/python-devicetree/tests/test_edtlib.py
@@ -126,7 +126,7 @@ def test_interrupts():
         edtlib.ControllerAndData(node=node, controller=controller_2, data={'one': 0, 'two': 0, 'three': 5}, name=None, basename=None)
     ]
 
-    node = edt.get_node("/interrupt-map-bitops-test/node@70000000E")
+    node = edt.get_node("/interrupt-map-bitops-test/node@70000000e")
     assert node.interrupts == [
         edtlib.ControllerAndData(node=node, controller=edt.get_node('/interrupt-map-bitops-test/controller'), data={'one': 3, 'two': 2}, name=None, basename=None)
     ]

